app/assets/javascripts/modules/LoadTime.js

```diff
@@ -0,0 +1,52 @@
+// The following script sends the page load time per page to GA
+
+moj.Modules.LoadTime = {
+
+	labels: {
+		excellent: 'Under 1 second (Excellent)',
+		veryGood: '1 to 2 seconds (Very good)',
+		acceptable: '2 to 3 seconds (Acceptable)',
+		improve: '3 to 5 seconds (Try improving)',
+		fix: '5 to 8 seconds (Needs fixing)'
+	},
+	category: 'Page load time',
+
+	init: function() {
+		window.onload = this.pageLoaded.bind(this);
+	},
+
+	pageLoaded: function() {
+		if(typeof 'ga' === 'undefined')return;
```

Comment:
Could we add some spacing between the ifs and returns? I think its a bit easier to read :) 

```
	if (typeof 'ga' === 'undefined') return;
```

Reply:
Done